styles/pages/settings/settings.scss

```diff
@@ -103,6 +103,15 @@ $page-width: 1920px - $rightnav-width;
 		border: 1px solid rgba(0, 0, 0, 0.5);
 		border-radius: 2px;
 	}
+	
+	&__sensitivity {
+		padding: 0 6px 0 12px;
+		margin-bottom: 4px;
+		
+		&--margin-left {
+			margin-left: 12px;
+		}
```

Comment:
I think it's best to get rid of this class, and just use a utility class (`ml-4`) is equivalent. Especially since this isn't actually proper BEM, since `--foo` means a modifier on an existing block/element, but you're using this classes on the outer `<Panel class="settings__sensitivity settings-enum__values">` and it's children. If you wanted to do properly in BEM you'd need to define a new block e.g. `settings-sensitivity` and do some element `settings-sensitivity__setting` or something. But just using `mb-4` is way simpler here.